src/dil_demo/erp.clj

```diff
@@ -68,8 +68,8 @@
   (fn delegation-wrapper [{:keys [client-data] :as req}]
     (let [{::store/keys [commands] :as res} (app req)
           trip (->> commands
-                    (filter #(= [:put! :trips] (take 2 %)))
-                    (map #(nth % 2))
+                    (filter #(= [:publish! :trips] (take 2 %)))
```

Comment:
hierboven spelling in de docstring fixen. "Create policies in AR when trip is created."